packages/inquirer-common/src/types.ts

```diff
@@ -0,0 +1,85 @@
+import { type IMessageSeverity, Severity } from '@sap-devx/yeoman-ui-types';
+import type {
+    Answers,
+    ConfirmQuestion as BaseConfirmQuestion,
+    InputQuestion as BaseInputQuestion,
+    ListQuestion as BaseListQuestion,
+    ListChoiceOptions,
+    PromptFunction,
+    PromptModule,
+    Question
+} from 'inquirer';
+
+// Re-export for convienient import in consuming modules
+export { type IMessageSeverity, Severity };
```

Comment:
I am not against convenience however, I haven't seen where it is used in the two consuming modules. I only saw it being used in tests. `@sap-devx/yeoman-ui-types` isn't a huge module, however, I wonder if this dependency is really required here if it is just used in tests.
If I missed another usage, or something else is planned, please let me know.

Reply:
Removed thanks.

Reply:
`@sap-devx/yeoman-ui-types` should become a `devDependency` now since there is only a `import type` usage left (as far as I saw).

Reply:
Done thx